### PR TITLE
Translations for examples

### PR DESCRIPTION
**Pull request by alygee**
Adding russian translations for code examples

## Description
close https://github.com/epicmaxco/vuestic-ui/issues/1000

Slightly corrected the translation in the code. Carried out the function to the helper. Now the translation is performed immediately after parsing the file.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/docs/src/examples/va-backtop/Default.vue [hunk 1]
```diff
@@ -5,7 +5,7 @@
       false-value="bottom"
       true-inner-label="top"
       false-inner-label="bottom"
-      label="Vertical Position:"
```
Comment (asvae) [inline packages/docs/src/examples/va-backtop/Default.vue:8]: I would argue translations are needed here.

File: packages/docs/src/examples/va-badge/WithCard.vue [hunk 2]
```diff
@@ -3,7 +3,7 @@
     <va-badge text="New" overlap>
       <va-card>
         <va-card-content>
-          Next update is right behind the corner
```
Comment (asvae) [inline packages/docs/src/examples/va-badge/WithCard.vue:6]: That's just generic text, you can add universal translation.

File: packages/docs/src/examples/va-card/Actions.vue [hunk 3]
```diff
@@ -2,19 +2,31 @@
   <div class="row row">
     <div class="flex md6 lg4">
       <va-card>
-        <va-card-title>Between horizontal alignment</va-card-title>
+        <va-card-title>
+          {{ $t('api.VaCard.examples.actions.firstTitle') }}
+        </va-card-title>
         <va-card-actions align="between">
-          <va-button>Action 1</va-button>
```
Comment (asvae) [inline packages/docs/src/examples/va-card/Actions.vue:7]: That's at best generic, or keep english.

File: packages/docs/src/examples/va-divider/Inset.vue [hunk 4]
```diff
@@ -2,12 +2,16 @@
   <div class="my-5 flex lg4">
     <p class="example-item">
       <va-icon name="home" />
-      <span class="float-right">I'm a list item</span>
```
Comment (asvae) [inline packages/docs/src/examples/va-divider/Inset.vue:5]: Most likely you can keep this text as generic.

File: packages/docs/src/examples/va-divider/Vertical.vue [hunk 5]
```diff
@@ -1,9 +1,15 @@
 <template>
   <div class="row my-5">
-    <span class="flex lg1">Title</span>
```
Comment (asvae) [inline packages/docs/src/examples/va-divider/Vertical.vue:3]: Could be moved to generic translations to simplify reuse.
mb fillerText or something.

File: packages/docs/src/examples/va-form/WithInputs.vue [hunk 6]
```diff
@@ -1,23 +1,24 @@
 <template>
   <div>
-    <va-button
-      class="mb-4 mr-4"
-      @click="$refs.form.validate()"
-    >
-      Validate
+    <va-button class="mb-4 mr-4" @click="$refs.form.validate()">
+      {{ $t('api.VaForm.examples.validate') }}
```
Comment (asvae) [inline packages/docs/src/examples/va-form/WithInputs.vue:4]: Should be generic.

File: packages/docs/src/examples/va-image/DefaultSlot.vue [hunk 7]
```diff
@@ -1,30 +1,27 @@
 <template>
   <div class="row">
     <div class="flex md6 lg4">
-      <va-image
-        style="min-height: 300px;"
-        src="https://picsum.photos/1500"
-      >
-        <va-badge text="default slot" />
+      <va-image style="min-height: 300px;" src="https://picsum.photos/1500">
+        <va-badge :text="$t('api.VaImage.examples.defaultSlot')" />
```
Comment (asvae) [inline packages/docs/src/examples/va-image/DefaultSlot.vue:5]: `default slot` you can leave the way it is. Hardly translatable.

File: packages/docs/src/examples/va-image/ErrorSlot.vue [hunk 8]
```diff
@@ -1,12 +1,9 @@
 <template>
   <div class="row">
     <div class="flex md6 lg4">
-      <va-image
-        style="min-height: 300px;"
-        src="wrong-image-path"
-      >
+      <va-image style="min-height: 300px;" src="wrong-image-path">
         <template #error>
-          Image not found! :(
```
Comment (asvae) [inline packages/docs/src/examples/va-image/ErrorSlot.vue:9]: You can replace with "Not found" and make generic.
Please remember, that every line you add has to be translated and supported.

File: packages/docs/src/examples/va-infinite-scroll/CustomTarget.vue [hunk 9]
```diff
@@ -42,19 +27,19 @@
...
     return {
       recordsRef: [{}, {}, {}, {}, {}, {}, {}, {}],
       recordsId: [{}, {}, {}, {}, {}, {}, {}, {}],
     }
   },
   methods: {
-    async appendRecordsAsyncRef () {
-      await new Promise(resolve => setTimeout(resolve, 1000))
+    async appendRecordsAsyncRef() {
+      await new Promise((resolve) => setTimeout(resolve, 1000))
```
Comment (asvae) [inline packages/docs/src/examples/va-infinite-scroll/CustomTarget.vue:38]: nit. We can make do without brackets:

File: packages/docs/src/examples/va-modal/Customization.vue [hunk 10]
```diff
@@ -1,22 +1,20 @@
...
+        <h2>
+          {{ $t('api.VaModal.examples.custom.header') }}
+        </h2>
       </template>
       <slot>
         <div>{{ message }}</div>
       </slot>
       <template #footer>
         <va-button>
-          Custom action
```
Comment (asvae) [inline packages/docs/src/examples/va-modal/Customization.vue:19]: Probably could also be generic.

File: packages/docs/src/examples/va-modal/HidingOverlay.vue [hunk 11]
```diff
@@ -1,20 +1,28 @@
 <template>
   <p>
-    <va-button class="mr-4" @click="showModal = !showModal">
-      Show modal (without the overlay)
```
Comment (asvae) [inline packages/docs/src/examples/va-modal/HidingOverlay.vue:4]: This one could probably be split into generic/non-generic (in brackets) parts.

File: packages/docs/src/examples/va-modal/ModalSizing.vue [hunk 12]
```diff
@@ -1,36 +1,28 @@
 <template>
   <div>
-    <va-button class="mr-4" @click="showModalSizeSmall = !showModalSizeSmall">
-      Show modal size small
```
Comment (asvae) [inline packages/docs/src/examples/va-modal/ModalSizing.vue:4]: Same here.

File: packages/docs/src/examples/va-backtop/Default.vue (hunk 1 above)
Comment (alygee) [reply packages/docs/src/examples/va-backtop/Default.vue:8]: I left the text in this place without translation

File: packages/docs/src/examples/va-badge/WithCard.vue (hunk 2 above)
Comment (alygee) [reply packages/docs/src/examples/va-badge/WithCard.vue:6]: moved the sentence to a generic text with universal translation

File: packages/docs/src/examples/va-card/Actions.vue (hunk 3 above)
Comment (alygee) [reply packages/docs/src/examples/va-card/Actions.vue:7]: moved to generic

File: packages/docs/src/examples/va-divider/Inset.vue (hunk 4 above)
Comment (alygee) [reply packages/docs/src/examples/va-divider/Inset.vue:5]: did the same

File: packages/docs/src/examples/va-divider/Vertical.vue (hunk 5 above)
Comment (alygee) [reply packages/docs/src/examples/va-divider/Vertical.vue:3]: moved to generic translations

File: packages/docs/src/examples/va-form/WithInputs.vue (hunk 6 above)
Comment (alygee) [reply packages/docs/src/examples/va-form/WithInputs.vue]: moved to generic translations

File: packages/docs/src/examples/va-image/DefaultSlot.vue (hunk 7 above)
Comment (alygee) [reply packages/docs/src/examples/va-image/DefaultSlot.vue]: left `default slot` without translation

File: packages/docs/src/examples/va-image/ErrorSlot.vue (hunk 8 above)
Comment (alygee) [reply packages/docs/src/examples/va-image/ErrorSlot.vue:9]: moved to generic translations

File: packages/docs/src/examples/va-infinite-scroll/CustomTarget.vue (hunk 9 above)
Comment (alygee) [reply packages/docs/src/examples/va-infinite-scroll/CustomTarget.vue]: ok, configured my formatter

File: packages/docs/src/examples/va-modal/Customization.vue (hunk 10 above)
Comment (alygee) [reply packages/docs/src/examples/va-modal/Customization.vue:19]: moved to generic translations

File: packages/docs/src/examples/va-modal/HidingOverlay.vue (hunk 11 above)
Comment (alygee) [reply packages/docs/src/examples/va-modal/HidingOverlay.vue:4]: the sentence separated and `Show modal` moved to generic translations

File: packages/docs/src/examples/va-modal/ModalSizing.vue (hunk 12 above)
Comment (alygee) [reply packages/docs/src/examples/va-modal/ModalSizing.vue:4]: did the same here and at other places

**Pull request closed without merging**